### ISSUE TITLE
Update hamburger-button

### ISSUE THREAD

**Issue by lwaldner** [funkhaus/fh-components#64]
Update hamburger button to allow for easy customizability. 

Possible solution would to have props that define things like fill color, width and distance between lines

**Comment by SaFrMo** [comment]
Great idea! Master to-do list:

~~~- [ ] Line color~~~
~~~- [ ] Stroke width~~~
~~~- [ ] Distance between lines~~~
- [ ] Allow complete override of button? ie a `slot` defaulting to a hamburger SVG but allowing any button content
~~~- [ ] Listener on click~~~

(Edit: I removed the items on the list other than the slot. We can attach a listener when the template is actually used and if a user wanted the other props, they're better off adding their own SVG!)

**Comment by SaFrMo** [comment]
Fixed in 1f7dad8 (version 1.2.79)!
